litestar/static_files/base.py

```diff
@@ -31,6 +32,7 @@ def __init__(
         file_system: FileSystemProtocol,
         send_as_attachment: bool = False,
         resolve_symlinks: bool = True,
+        cache_control: CacheControlHeader | None = None,
```

Comment:
I think accepting `headers` here directly and resolving the `CacheControlHeader` on the layer above if present makes more sense

Reply:
Yeah, that's much cleaner.